lib/chime-sdk-stack.ts

```diff
@@ -27,21 +21,21 @@ export class ChimeSdkStack extends NestedStack {
     this.phoneNumber = phoneNumber.phoneNumber;
 
     const sipMediaApp = new chime.ChimeSipMediaApp(this, 'sipMediaApp', {
-      region: this.region,
-      endpoint: props?.smaLambdaEndpointArn as string,
+      region: Stack.of(this).region,
+      endpoint: props.smaLambdaEndpointArn as string,
```

Comment:
is the smaLambdaEndpointArn presence actually guaranteed? What behavior do you want if that prop doesn't exist?

Reply:
Yes, for certain.  This stack will fail entirely without that.  There is no recovery.